opensearch_py_ml/ml_commons/ml_common_client.py

```diff
@@ -77,3 +78,56 @@ def load_model(self, model_id: str):  # type: ignore
             method="POST",
             url=API_URL,
         )
+
+    def get_task_info(self, task_id: str):  # type: ignore
+        """
+        This method return information about a task running into opensearch cluster (using ml commons api)
+        when we load a model.
+
+        Parameters
+        ----------
+        :param task_id: unique id of the task
+        :type task_id: string
+
+        Returns
+        -------
+        :return: returns a json object, with detailed information about the task
+        :rtype: object
+        """
+
+        MODEL_TASK_API_ENDPOINT = f"tasks/{task_id}"
+        API_URL = f"{ML_BASE_URI}/{MODEL_TASK_API_ENDPOINT}"
+
+        return self._client.transport.perform_request(
+            method="GET",
+            url=API_URL,
+        )
+
+    def generate_embedding(self, model_id: str, sentences: List[str]):  # type: ignore
```

Comment:
Not for this line. 
How about adding methods for all other REST APIs like `unload`, `delete model`, `get model` etc.?

Reply:
Good idea. Can I add these in a separate PR?

Reply:
Sure, that's ok

Reply:
Changed my mind, added all these 3 with integration test and documentation as well.